client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -121,24 +134,37 @@ class PacmanGame extends Component {
 
   render() {
     const { gridSize, gridState, pacman } = this.state;
+    const { classes } = this.props;
+
     return (
-      <div>
-        <button type="button" className="" onClick={this.startGame}>
-          Start
-        </button>
-        <PacmanBoard
-          gridSize={gridSize}
-          gridState={gridState}
-          pacman={pacman}
-        />
-      </div>
+      <React.Fragment>
+        <Paper className={classes.root} elevation={1}>
+          <Grid container spacing={16}>
+            <Grid item xs={3} />
+            <Grid item xs={6}>
+              <button type="button" className="" onClick={this.startGame}>
```

Comment:
Create this as a separate component, as there will be several buttons in thr app, to create consistency among buttons this will be helpful